src/style.css

```diff
@@ -1 +1,107 @@
 /* add styles here */
+@font-face {
+  font-family: Avenir;
+  src: url("./fonts/Avenir-Book.ttf") format("truetype");
```

Comment:
```suggestion
  src: url("./fonts/Avenir-Book.ttf");
```

Reply:
Another mentor in chat sent me this example, so I don't really understand which variant is right

Reply:
you may google what does it mean

generally, it's needed to explicitly set the format of the font file:
https://css-tricks.com/snippets/css/using-font-face-in-css/

but it could work w/o it

so you may keep any variant